setup.cfg

```diff
@@ -1,2 +1,2 @@
 [flake8]
-max-line-length = 99
+max-line-length = 105
```

Comment:
Rather then increasing the max allowed line length for flake8, should we define a common value that is max line length for both isort and flake8?

Reply:
Updated accordingly!